src/typography/brand/Brand.style.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+import { color, font, fontWeight } from '../../_utils/branding'
+import { Text } from '../index'
+
+export const TextBrand = styled(Text)`
```

Comment:
Shouldn't we be more semantic here and use a heading?

Reply:
We can, doing `<TextBrand as="h1">` as for other text components

Reply:
Yep I agree however I was thinking of enforcing it so that we don't forget when using it in Kairos.

Reply:
I see what you mean. But if we do so, it should in `BrandedVoice`, not the typography element